alignak/daemons/receiverdaemon.py

```diff
@@ -222,6 +223,9 @@ def setup_new_conf(self):
             self.direct_routing = conf['global']['direct_routing']
             self.accept_passive_unknown_check_results = \
                 conf['global']['accept_passive_unknown_check_results']
+            # Update External Commands Manager
+            self.external_commands_manager.accept_passive_unknown_check_results = \
```

Comment:
No, alerting the end-user that some checks were received in the wrong place ... for instance in a realm configuration